etc/helm/pachyderm/templates/pachd/config-secret.yaml

```diff
@@ -2,7 +2,7 @@
 SPDX-FileCopyrightText: Pachyderm, Inc. <info@pachyderm.com>
 SPDX-License-Identifier: Apache-2.0
 */ -}}
-{{ if .Values.pachd.enterpriseLicenseKey }}
+{{ if .Values.pachd.activateEnterprise }}
```

Comment:
I like changing this to be an explicit bool instead of the implicit value check on license key

Reply:
that sounds good to me. I was conflicted on this and made the initial decision, based on reducing parameters. I agree being explicit is something we should strive for in configuration